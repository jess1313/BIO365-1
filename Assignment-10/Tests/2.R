suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("2.R"))))

numVector1 = 1:50
numVector2 = rep(c(1:5), 50)
set.seed(1)
numVector3 = rnorm(1000)

print(round(calcVariance(numVector1), 8))
print(round(calcVariance(numVector2), 8))
print(round(calcVariance(numVector3), 8))