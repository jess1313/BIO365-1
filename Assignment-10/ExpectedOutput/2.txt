[1] 212.5
[1] 2.008032
[1] 1.071051
